src/OnboardingSPA/components/MiniPreview/index.js

```diff
@@ -72,9 +72,12 @@ const MiniPreview = ({ title, desc, icon, socialData }) => {
     function socialIconList() {
         var socialIconList = []
         socialDataset.map( (socialInfo) => {
-                !socialInfo.url ||
-                socialIconList.push(
-                    <div key={socialInfo.image} className={`browser-content_social_icon ${isValidUrl(socialInfo.url) || 'invalid-url' }`} style={{ backgroundImage: socialInfo.image }} />)
+            socialIconList.push(
+                <div key={socialInfo.image} 
+                    onClick={(e) => setIsSocialFormOpen(!isSocialFormOpen)}
+                    className={`browser-content_social_icon ${socialInfo.url ? isValidUrl(socialInfo.url) || '--invalid-url' : '--no-url' }`} 
+                    style={{ backgroundImage: socialInfo.image }} />
+            )
```

Comment:
We can use `map` here Ref comment on https://github.com/newfold-labs/wp-module-onboarding/pull/138